homeassistant/components/advantage_air/binary_sensor.py

```diff
@@ -56,15 +54,13 @@ class AdvantageAirZoneMotion(AdvantageAirEntity, BinarySensorEntity):
 
     _attr_device_class = DEVICE_CLASS_MOTION
 
-    @property
-    def name(self):
-        """Return the name."""
-        return f'{self._zone["name"]} Motion'
-
-    @property
-    def unique_id(self):
-        """Return a unique id."""
-        return f'{self.coordinator.data["system"]["rid"]}-{self.ac_key}-{self.zone_key}-motion'
+    def __init__(self, instance, ac_key, zone_key):
+        """Initialize an Advantage Air Zone Motion."""
+        super().__init__(instance, ac_key, zone_key)
+        self._attr_name = f'{self._zone["name"]} Motion'
```

Comment:
This appears to prevent the name from being updated by the source without a restart.

Reply:
See discussion above.